_quarto-development.yml

```diff
@@ -0,0 +1,8 @@
+website:
+  title: Biomarker Analysis Catalog - Dev
```

Comment:
@danielinteractive Are these titles okay? Or should `- Dev` and `- Stable` be changed to something else?

Reply:
yeah those are fine